src/Sulu/Bundle/HttpCacheBundle/Tests/Unit/DependencyInjection/SuluHttpCacheExtensionTest.php

```diff
@@ -71,6 +77,12 @@ public function setUp(): void
         $this->container->set('sulu_core.webspace.webspace_manager.url_replacer', $this->replacer->reveal());
         $this->container->set('logger', $this->logger->reveal());
         $this->container->set('sulu_website.reference_store_pool', $this->referenceStore->reveal());
+        $this->container->set('sulu.content.structure_manager', $this->prophesize(StructureManagerInterface::class)->reveal());
+        $this->container->set('sulu_document_manager.document_inspector', $this->prophesize(DocumentInspector::class)->reveal());
+        $this->container->set('sulu.content.resource_locator.strategy_pool', $this->prophesize(ResourceLocatorStrategyPoolInterface::class)->reveal());
+        $this->container->set('sulu_tag.tag_manager', $this->prophesize(TagManagerInterface::class)->reveal());
+        $this->container->set('fos_http_cache.cache_manager', $this->prophesize(CacheManager::class)->reveal());
+        $this->container->set('fos_http_cache.http.symfony_response_tagger', $this->prophesize(SymfonyResponseTagger::class)->reveal());
```

Comment:
Not sure why this only happens on 4.4-dev but this are requirements of:

https://github.com/sulu/sulu/blob/2.0.0/src/Sulu/Bundle/HttpCacheBundle/Resources/config/event-subscribers.xml#L6-L17

Reply:
@danrot this seems to be needed also for the 4.4 tests now